packages/fxa-auth-server/config/index.js

```diff
@@ -569,6 +569,54 @@ const conf = convict({
     env: 'DEVICE_NOTIFICATIONS_ENABLED',
     default: true
   },
+  subhub: {
+    enabled: {
+      doc: 'Indicates whether talking to the SubHub server is enabled',
+      format: Boolean,
+      default: false,
```

Comment:
Should the default value for this be `true`, with the `false` override in `dev.json`? I think Ops have a preference that the production config uses as many defaults as possible.

Reply:
I made the default false since we are likely to merge & deploy this stuff in advance of any subhub installations being available. Figured that would be safer, but maybe we can ensure it's false along with the deployment